### ISSUE TITLE
[Performance] After clicking on a search result it took 4x OpenReport requests to open the final report

### ISSUE THREAD

**Issue by hannojg** [Expensify/App#46594]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## What performance issue do we need to solve?

A customer was searching for a chat, clicking on it, and then it took a long loading time and actually 4x `OpenReport` queries for the final chat to open:

https://github.com/user-attachments/assets/53bd9d0b-13ca-477e-908f-7b99cd346a2e

## What is the impact of this on end-users?

TTI with the chat

## List any benchmarks that show the severity of the issue

I have a copy of all network requests available. As they contain customer data I can't post them here. If you're working on that issue please let me know

The customer also  shared a profile trace with us:

[Firefox 2024-07-25 10.42 profile.json.gz](https://github.com/user-attachments/files/16443244/Firefox.2024-07-25.10.42.profile.json.gz)

(note: the trace also contains other test cases as well)

## Proposed solution (if any)

None yet, I will go through the profile and see what can be optimised, what exactly caused those lags.

## List any benchmarks after implementing the changes to show impacts of the proposed solution (if any)

not available yet 

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

**Version Number:** v9.0.11-5
**Reproducible in staging?:** not tested
**Reproducible in production?:** yes
**Email or phone of affected tester (no customers):** customer
**Logs:** See performance file
**Notes/Photos/Videos:** See attached video
**Expensify/Expensify Issue URL:** n/a
**Issue reported by:** @hannojg
**Slack conversation:** https://expensify.slack.com/archives/C05LX9D6E07/p1721919928992729

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

**Comment by hannojg** [comment]
cc @sakluger (feel free to assign me as I (or someone from my team) will work on this ticket!)

**Comment by sakluger** [comment]
No updates

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
@hannojg should we leave this as a daily, or will it be a week or more before you can prioritize this one?

**Comment by sakluger** [comment]
Moving to weekly for now since Hanno is out until Monday.

**Comment by sakluger** [comment]
@hannojg any updates on this one?

**Comment by hannojg** [comment]
This is a rather tricky bug as reproducing it is hard. We have this currently as low priority to address overall performance issues first!

**Comment by sakluger** [comment]
I asked for updates in the main tracking issue: https://github.com/Expensify/App/issues/46595

**Comment by sakluger** [comment]
We're focusing on the slow search first. We have a WIP PR for that, I assume we'll look at this again once that is done.

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
No update.

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
No updates

**Comment by sakluger** [comment]
No updates

**Comment by sakluger** [comment]
No updates

**Comment by sakluger** [comment]
No updates.

**Comment by sakluger** [comment]
No update.

**Comment by sakluger** [comment]
No update.

**Comment by sakluger** [comment]
No update

**Comment by sakluger** [comment]
@hannojg what's the latest on this one? Are you working on it, or is it on hold for other priorities/investigations?

**Comment by sakluger** [comment]
No updates - I'll ask in Slack later this week or next week for the status of the general investigation.
